### PR TITLE
Bump deck.gl to 8.7.3

### PR DESCRIPTION
**Pull request by bbecquet**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,13 +5,14 @@
...
   ],
   "devDependencies": {
-    "@deck.gl/carto": "^8.7.0",
-    "@deck.gl/core": "^8.7.0",
-    "@deck.gl/extensions": "^8.7.0",
-    "@deck.gl/geo-layers": "^8.7.0",
-    "@deck.gl/google-maps": "^8.7.0",
-    "@deck.gl/layers": "^8.7.0",
-    "@deck.gl/mesh-layers": "^8.7.0",
+    "@deck.gl/aggregation-layers": "^8.7.2",
```
Comment (bbecquet) [inline package.json:8]: I had to add this new dependency for the build to pass. Is it normal?

Comment (Clebal) [reply package.json:8]: image

@deck.gl/aggregation-layers is a new peer-dependencies of @deck.gl/carto

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Not released
 
+- Bump deck.gl to 8.7.0 [#360](https://github.com/CartoDB/carto-react/pull/360)
```
Comment (Clebal) [inline CHANGELOG.md:5]: 8.7.2, right?

**Review by Clebal — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (bbecquet) [reply CHANGELOG.md:5]: Oops. Thank you!

File: package.json (hunk 1 above)
Comment (bbecquet) [reply package.json:8]: Yes, Felix added it after my comment :)
That's why I'm actually waiting for 8.7.3 to be released to update directly to that version.

Comment (Clebal) [reply package.json:8]: Great 👌🏻

Comment (bbecquet) [reply package.json]: @Clebal I've updated to 8.7.3

**Review by Clebal — APPROVED** [review]

**Pull request merged**